Diagnostics/start-windows-build.ps1

```diff
@@ -0,0 +1,388 @@
+Param(
+    [Parameter(Mandatory=$false)]
+    [string]$GitURL="https://github.com/dcos/dcos-diagnostics", 
+    [Parameter(Mandatory=$false)]
+    [string]$Branch="master",
+    [Parameter(Mandatory=$false)]
+    [string]$CommitID,
+    [Parameter(Mandatory=$false)]
+    [string]$ParametersFile="${env:WORKSPACE}\build-parameters.json",
+    [Parameter(Mandatory=$false)]
+    [switch]$EnableSSL
```

Comment:
Also, I forgot about this one. We can remove the parameter switch `$EnableSSL`. This was used for the Mesos script.